packages/components/src/molecules/Rating/Rating.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { forwardRef, useState } from 'react'
+import type { ReactNode } from 'react'
+import type { HTMLAttributes } from 'react'
+
+import List from '../../atoms/List'
+import { Icon, IconButton } from '../..'
+
+export interface RatingProps extends HTMLAttributes<HTMLUListElement> {
+  /**
+   * The current value of the rating, based on the quantity of child elements.
+   */
+  value: number
+  /**
+   * A React component that will be rendered as an icon.
+   */
+  icon: ReactNode
+  /**
+   * Specifies that this input should be controled by the user.
+   */
+  actionable?: boolean
```

Comment:
I think this prop should be replaced by `onChange` because this rating is `actionable` for a reason, which is to get the rate that the user voted for. Using this boolean, it's not possible, using `onChange` as a function, you can check if the component must be actionable and provide a way for the developer to get the result.

Reply:
Agreed!